cpu/stm32/Makefile.include

```diff
@@ -55,9 +60,11 @@ $(call target-export-variables,$(VECTORS_O),CPU_LINE)
 # which are only used for STM32 system includes and not of interest for RIOT.
 CFLAGS += -D__SYSTEM_STM32$(call uppercase,$(CPU_FAM))XX_H
 
-STM32CMSIS_INCLUDE_DIR = $(RIOTCPU)/stm32/include/vendor/cmsis/$(CPU_FAM)/Include
-STM32FAM_INCLUDE_FILE = $(STM32CMSIS_INCLUDE_DIR)/stm32$(CPU_FAM)xx.h
-INCLUDES += -I$(STM32CMSIS_INCLUDE_DIR)
+ifeq (, $(filter STM32MP157Cxx, $(CPU_LINE)))
```

Comment:
```suggestion
ifeq (,$(filter STM32MP157Cxx,$(CPU_LINE)))
```
Also add a comment why this is needed because this is not obvious for the uninitiated.